bin/hdfcoinc/pycbc_multiifo_dtphase

```diff
@@ -0,0 +1,169 @@
+#!/bin/env python
+""" Create a file containing the phase and amplitude, 
+correlations between two detectors by
+doing a simple monte-carlo
+
+Output is the relative SNR, time, and phase as compared to a reference
+IFO (the first one provided). 
+
+To get the noise rate this should be scaled by the local sensitivity value
+and by the SNR of the signal in the reference detector 
+(see pycbc/events/stat.py)
```

Comment:
this is too vague, stat.py is 700 lines long. name a specific class or (better) function?